pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -132,7 +132,11 @@ func removePodAllSecondaryNetwork(podCNIInfo *cnipodcache.CNIConfigInfo) error {
 	// Clean-up IPAM at whereabouts db (etcd or kubernetes API server) for all the secondary networks of the Pod which is getting removed.
 	// NOTE: SR-IOV VF interface clean-up, upon Pod delete will be handled by SR-IOV device plugin. Not handled here.
 	cmdArgs = &invoke.Args{Command: string("DEL"), ContainerID: podCNIInfo.ContainerID,
-		NetNS: podCNIInfo.ContainerNetNS, Path: cniPath}
+		NetNS: podCNIInfo.ContainerNetNS, Path: cniPath, PluginArgs: [][2]string{
+			{"K8S_POD_NAME", podCNIInfo.PodName},
+			{"K8S_POD_NAMESPACE", podCNIInfo.PodNameSpace},
+			{"K8S_POD_INFRA_CONTAINER_ID", podCNIInfo.ContainerID},
+		}}
```

Comment:
maybe add a comment to explain that these variables are required by the version of whereabouts we are using

Reply:
also just to confirm: whereabouts expect these args for DEL and not just for ADD?

Reply:
yes.